### PR TITLE
Update both Python and Ruby dependencies

### PR DESCRIPTION
**Pull request by TrueBrain**

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -17,13 +17,10 @@ plugins:
 exclude:
   - Dockerfile
   - fetch-downloads.py
-  - Gemfile
-  - Gemfile.lock
   - nginx.default.conf
   - requirements.base
   - requirements.txt
   - thirdparty
-  - vendor
```
Comment (LordAro) [inline _config.yml:26]: these excludes no longer necessary or something? Why removed?

File: Dockerfile [hunk 2]
```diff
@@ -25,15 +25,14 @@ COPY Gemfile Gemfile.lock /source/
 
 RUN apk --no-cache add \
         build-base \
+        libstdc++ \
```
Comment (LordAro) [inline Dockerfile:28]: I'm curious what (newly) depends on this

File: _config.yml (hunk 1 above)
Comment (TrueBrain) [reply _config.yml:26]: https://jekyllrb.com/docs/upgrading/3-to-4/#exclusion-changes

File: Dockerfile (hunk 2 above)
Comment (TrueBrain) [reply Dockerfile:28]: libsaas, as it turns out.

**Review by LordAro — APPROVED** [review]

**Pull request merged**